src/components/composer/composer/composer.ts

```diff
@@ -57,6 +57,7 @@ export const tokenColors = {
 css/* scss */ `
   .o-composer-container {
     .o-composer {
+      font-family: ${DEFAULT_FONT};
```

Comment:
It defies the whole purpose of not forcing any font (to speed up the first page load). But I honestly don't care because I think it's stupidly overkill as an optimisation.
(having the same font in as odoo in odoo is not stupid though)

Reply:
The font will need to be loaded anyway, since it's the font we use un the canvas. And I _really_ don't think we want a System depandant font in the canvas.